.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,25 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install --upgrade pip wheel
+      - run: pip install bandit codespell darker flake8 flake8-2020 flake8-bugbear
+                         flake8-comprehensions isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101 . || true  # B101 is assert statements
+      - run: darker --check .
```

Comment:
We already run Darker itself on its own code base using [pytest-darker](https://pypi.org/project/pytest-darker/), see
- [`.github/workflows/python-pacakge.yml`](https://github.com/akaihola/darker/blob/8a1835c952b9743ea5cb6d40d23eb9c6458e7828/.github/workflows/python-package.yml#L81-L83)
- [`setup.cfg`](https://github.com/akaihola/darker/blob/8a1835c952b9743ea5cb6d40d23eb9c6458e7828/setup.cfg#L58)
- [`pytest.ini`](https://github.com/akaihola/darker/blob/8a1835c952b9743ea5cb6d40d23eb9c6458e7828/pytest.ini#L5)

I agree it might be a better idea to run Darker explicitly instead of through [pytest-darker](https://pypi.org/project/pytest-darker/). In that case, the PR that makes the change should also remove `--darker` from [`pytest.ini`](https://github.com/akaihola/darker/blob/8a1835c952b9743ea5cb6d40d23eb9c6458e7828/pytest.ini#L5).

Reply:
I wrote:
>I agree it might be a better idea to run Darker explicitly instead of through [pytest-darker](https://pypi.org/project/pytest-darker/)

On a second thought, currently the Darker repository also serves as a "test case" for [pytest-darker](https://pypi.org/project/pytest-darker/). Removing the use of `pytest-darker` would
1. prevent us from seeing when we break compatibility between Darker and `pytest-darker`
2. fail to promote `pytest-darker` for the curious eyes who take a look at Darker's workflows